frontend/src/views/Shares/ShareView.js

```diff
@@ -140,14 +151,16 @@ function ShareViewHeader(props) {
     }
     setAccepting(false);
   };
```

Comment:
Since it is a free text field, users might add long descriptions that surpass the memory limits of an RDS cell. In the creation forms (CreateDataset...) we limit the description fields to 5000 chars (`frontend/src/views/Environments/EnvironmentCreateForm.js:375`). We can access the actual limits in the RDS database parameters, but I would say that in any case we need to do a check before writing to RDS

Reply:
Good call out - let me add a string length limit for the reasons provided

Reply:
Added this character limit